RecoJets/JetProducers/src/PileupJetIdAlgo.cc

```diff
@@ -391,12 +391,21 @@ PileupJetIdentifier PileupJetIdAlgo::computeIdVariables(const reco::Jet * jet, f
 			  }
 			  if(pfTrk==nullptr) { //protection against empty pointers for the miniAOD case
 			    //To handle the electron case
-			    if(lPF!=nullptr) {
+			    if(isPacked) {
```

Comment:
given that #20491 is merged, you could use the `PackedCandidate::dz(point)` and `PackedCandidate::dxy(point)` methods, which are available for all particles (`hasTrackDetails` is only needed to get the covariance, hitpattern or other more detailed information)

Reply:
ok, I will change this. Was #20491 also backported to 93?